app/src/main/java/org/fossasia/phimpme/utilities/Constants.java

```diff
@@ -26,5 +26,8 @@
 
     public static String IMGUR_HEADER_CLIENt = "Client-ID";
     public static String IMGUR_HEADER_USER = "Bearer";
+
+    public final static String APP_KEY = "9qgi9rcomdglx8f";
```

Comment:
@the-dagger Made the required changes, added the description to configure the app keys in the README.md file.